src/components/RNTextInput.tsx

```diff
@@ -0,0 +1,29 @@
+import React, {ForwardedRef} from 'react';
+// eslint-disable-next-line no-restricted-imports
+import {TextInput, TextInputProps} from 'react-native';
+import Animated, {AnimateProps} from 'react-native-reanimated';
```

Comment:
```suggestion
import Animated, {AnimatedProps} from 'react-native-reanimated';
```

`AnimateProps` is deprecated 🙂

Reply:
This is a type that `AnimatedTextInput` is using, would you change it anyways?

Reply:
`AnimateProps` is an alias for `AnimatedProps` as we can see [here](https://github.com/software-mansion/react-native-reanimated/blob/main/src/reanimated2/helperTypes.ts#L179), so it should not be a problem to use `AnimatedProps` 🙂